CMakeLists.txt

```diff
@@ -992,3 +997,8 @@ list(APPEND CPACK_SOURCE_IGNORE_FILES "${CMAKE_BINARY_DIR}")
 # CPack must be included after the CPACK_* variables are set in order for those
 # variables to take effect.
 Include(CPack)
+
+if(_M_X86_64)
```

Comment:
This isn't needed in x86_64 if built generic (Which my PR makes note of without explicitly doing so)